src/types/message.ts

```diff
@@ -1,89 +1,103 @@
-export type Details = 'basic' | 'tokens' | 'tokenBalances' | 'txids' | 'txs';
+export const Details = ['basic', 'tokens', 'tokenBalances', 'txids', 'txs'] as const;
```

Comment:
This is a var not a type, as such imo it should be in lowercase

Reply:
That's a common practice to use `Details` both as a value or as a type depending on the need.